source/partials/next_release/_release.md.erb

```diff
@@ -1,87 +1,88 @@
+<h4>Bug Fixes</h4>
 
-    Below is the list of plugins that will stop working in 17.1.
+* <%= link_to_commit '9cec83fd6dda71ea085525354ca201ab7e4d3dec', 'Stop very large file systems (EFS) from being identified as having no space.', 3426 %>
 
-    **Note:** The GoCD team has built a [shim](https://github.com/aresok/go-maven-poller/compare/master...arvindsv:make-it-work-with-gocd-17-1) that allows migration of all plugins with minimal change. We are actively working with plugin authors to ensure that these plugins are migrated over in time for 17.1.
+  Large filesystems like EFS, have an amount of storage that exceeds what can be addressed with a java `long` type (<code>2<sup>63</sup>-1</code>). Any storage space greater than <code>2<sup>63</sup>-1</code>, will be shown to be <code>2<sup>63</sup>-1</code>.
 
-    **Package plugins**
+<h4>Breaking Changes</h4>
 
-    * [Maven (Nexus) repository poller](https://github.com/aresok/go-maven-poller)
-    * [Docker Registry Poller - 1](https://github.com/hammerdr/go-docker-registry-poller)
-    * [Docker Registry Poller - 2](https://github.com/decoomanj/gocd-docker-material-poller)
-    * [Puppet Forge repository poller](https://github.com/drrb/go-puppet-forge-poller)
-    * [Debian repository poller](https://github.com/gocd-contrib/deb-repo-poller)
-    * [Cloud Foundry poller](https://github.com/Sounie/springer-gocd-cloudfoundry-plugin/blob/master/README.md)
+* <%= link_to_commit 'bd64d11d6d602ca4b0ab2318a3d504d07b472346', 'Removed Templates API v2', 3441 %>
 
-    **Task plugins**
+  Version 2 of the <%= link_to_api 'template-config', 'Templates API' %> was deprecated in 17.1. Version 3 of the API is available, and users are encouraged to use it.
 
-    * [MSBuild Task](https://github.com/akanyer/gocd-msbuild-taskplugin/blob/master/README.md)
-    * [XUnit Converter](https://github.com/gocd-contrib/xunit-converter-task)
-    * [Publish to S3](https://github.com/indix-eng/gocd-s3-artifacts)
-    * [Fetch from S3](https://github.com/indix-eng/gocd-s3-artifacts)
-    * [Powershell Task](https://github.com/manojlds/gocd-powershell-runner/blob/master/README.md)
-    * [RapidDeploy Package Builder](https://github.com/MidVision/go-rapiddeploy/wiki#rapiddeploy-package-repository-plugin)
-    * [RapidDeploy Job Runner](https://github.com/MidVision/go-rapiddeploy/wiki#rapiddeploy-job-runner-task-plugin)
-    * [Maven Task Plugin](https://github.com/ruckc/gocd-maven-plugin)
-    * [Publish to Artifactory](https://github.com/tusharm/go-artifactory-plugin)
+<h4>Deprecations</h4>
+
+Features that become superseded or irrelevant become *deprecated*, and are scheduled to be removed in some timeframe.
+
+The following features have been deprecated, along with the timeframe when they will be removed.
```

Comment:
Consider changing to: The following features have been deprecated. Please note the timeframe of when they will be removed.